lambdatimes/src/components/Content/Content.js

```diff
@@ -48,7 +63,7 @@ export default class Content extends Component {
           `selectedTab` that includes the currently selected tab
           and `selectTabHandler` that includes the function to change the selected tab
         */}
-        <Tabs tabs={this.state.tabs} />
+        <Tabs tabs={this.state.tabs} selectedTab={this.state.selected} changeSelected={this.changeSelected} />
```

Comment:
Possible refactor -
"selectedTab={this,state.selected}" -> "selected={this.state.selected}"